pkg/scan/process_image.go

```diff
@@ -213,6 +213,7 @@ func isExecutable(path string) bool {
 }
 
 func ScanFile(s *Scanner, f *os.File, iocs *[]output.IOCFound, layer string) error {
+	logrus.Infof("Scanning file %s", f.Name())
```

Comment:
I am printing this in info logs to be little verbose, let me know if we should move this to debug

Reply:
I think debug is better for this, yes